app/assets/javascripts/recourse.js

```diff
@@ -37,7 +37,10 @@ angular
 
 // Post Factory Function
 function PostFactoryFunction($resource) {
+
+  // Route to API for ngResource
   return $resource("/api/posts/:id.json", {}, {
+    create: { method: "POST" },
```

Comment:
Doesn't it do this by default? I'm still not clear on exactly how this worked for update though